chain/client/src/sync_jobs_actor.rs

```diff
@@ -149,11 +153,17 @@ impl actix::Handler<WithSpanContext<StateSplitRequest>> for SyncJobsActor {
     fn handle(
         &mut self,
         msg: WithSpanContext<StateSplitRequest>,
-        _: &mut Self::Context,
+        context: &mut Self::Context,
     ) -> Self::Result {
-        let (_span, msg) = handler_debug_span!(target: "client", msg);
-        tracing::debug!(target: "client", ?msg);
-        let response = Chain::build_state_for_split_shards(msg);
-        self.client_addr.do_send(response.with_span_context());
+        let (_span, state_split_request) = handler_debug_span!(target: "client", msg);
```

Comment:
Ideally we should have fallback system to not poll indefinitely. Maybe something like max poll time, after which we just fail.

Reply:
Added in some kind of max timeout check